Sources/ConsoleKit/Utilities/ConsoleLogger.swift

```diff
@@ -12,6 +12,8 @@ public struct ConsoleLogger: LogHandler {
     
     /// The conosle that the messages will get logged to.
     public let console: Console
+
+    private let sourcePathDelimiter: Substring
```

Comment:
What's the intended use case behind this change?

Reply:
It's to enable testing of the source path: https://github.com/vapor/console-kit/pull/149/files#diff-7648f81b352c42506685f0d7165e438eR68-R76

Also nice to have should someone decide not to use the `Sources` directory structure.

Reply:
Ah I see. I'm less sure about this change. It took me a bit to understand what the parameter actually did. 

What about having the logger split on both `Sources` and `Tests` by default? That way we get cleaner logs without the user needing to configure anything.

Reply:
Done.